OMCompiler/SimulationRuntime/c/util/simulation_options.c

```diff
@@ -545,6 +567,28 @@ const char *FLAG_DETAILED_DESC[FLAG_MAX+1] = {
   "  Run the Data Reconciliation numerical computation algorithm for constrained equations",
   /* FLAG_DATA_RECONCILE_BOUNDARY */
   "  Run the Data Reconciliation numerical computation algorithm for boundary condition equations",
+  /* FLAG_SR */
```

Comment:
The detailed description should be more detailed than the short description.
State the allowed flag values here and explain what inner and outer means, what the extrapolation is used for and so on.

Also check if the default values are still correct.